kamon-statsd/src/main/resources/reference.conf

```diff
@@ -5,18 +5,10 @@
 kamon {
   statsd {
 
-    # Hostname and port in which your StatsD is running. Remember that StatsD packets are sent using UDP and
-    # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
-    hostname = "127.0.0.1"
-    port = 8125
```

Comment:
top level only, please :). `max-packet-size` is specific to the batch metric sender, so it can remain where it is now.. thanks!